app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaHeaderHolder.kt

```diff
@@ -315,6 +312,28 @@ class MangaHeaderHolder(
         updateCover(manga)
     }
 
+    private fun setGenreTags(binding: MangaHeaderItemBinding, manga: Manga) {
+        with(binding.mangaGenresTags) {
+            removeAllViews()
+            if (manga.genre.isNullOrBlank().not()) {
+                (manga.getGenres() ?: emptyList()).map { genreText ->
+                    val chip = LayoutInflater.from(binding.root.context).inflate(
+                        R.layout.genre_chip,
+                        this,
+                        false
+                    ) as Chip
+                    val id = View.generateViewId()
+                    chip.id = id
+                    chip.text = genreText
```

Comment:
gonna add trim back to this because sources not named mangadex suck at this

Reply:
Pretty sure that method has trim?  It's not the accessor

Reply:
you're completely right